install/0000_80_machine-config-operator_01_config.crd.yaml

```diff
@@ -0,0 +1,192 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Is this file only here for testing purposes?

Reply:
I want to bring this into the MCO as preparatory work for future state reporting. It is harmless though since we do not act upon it.